server/plugin/taskLinkPreview.go

```diff
@@ -159,3 +159,44 @@ func (p *Plugin) PostBuildDetailsPreview(linkData []string, link, userID, channe
 	model.ParseSlackAttachment(post, []*model.SlackAttachment{attachment})
 	return post, ""
 }
+
+func (p *Plugin) PostReleaseDetailsPreview(linkData []string, link, userID, channelID string) (*model.Post, string) {
+	organization := linkData[3]
+	project := linkData[4]
+	releaseID := strings.Split(linkData[5], "&")[1][10:]
+	releaseDetails, _, err := p.Client.GetReleaseDetails(organization, project, releaseID, userID)
+	if err != nil {
+		return nil, ""
+	}
+
+	post := &model.Post{
+		UserId:    userID,
+		ChannelId: channelID,
+	}
+
+	environments := p.getPipelineReleaseEnvironmentList(releaseDetails.Environments)
+	attachment := &model.SlackAttachment{
+		AuthorName: "Azure Pipeline",
+		AuthorIcon: fmt.Sprintf("%s/plugins/%s/static/%s", p.GetSiteURL(), constants.PluginID, constants.FileNamePipelinesIcon),
```

Comment:
I think we should use the `public` folder for hosting all the icons and files.

Reply:
Yes, have already logged this here
https://github.com/mattermost/mattermost-plugin-azure-devops/pull/104#discussion_r1048073930